src/main/resources/xml/xslt/split-html.xsl

```diff
@@ -33,7 +33,7 @@
                     <xsl:copy-of select="/*/@* | @xml:base" exclude-result-prefixes="#all"/>
                     <xsl:namespace name="epub" select="'http://www.idpf.org/2007/ops'"/>
                     <xsl:text>
-</xsl:text>
+                    </xsl:text>
```

Comment:
Presumably this is not the same as before. The old code inserts a newline and no spaces. The new code inserts a newline and some spaces.

I tried the test suite and it passes. Are these additional spaces harmless?

Reply:
Good point. It seems harmless, but the intention is to only insert a newline, so we should fix this. Maybe we could add a `xml:space="preserve"` on the `xsl:text` if EditorConfig respects that. Alternatively we could say `<xsl:text>&#x0A;</xsl:text>` or `<xsl:value-of select="'\n'"/>`or similar.

Reply:
newline issue here

Reply:
> add a `xml:space="preserve"` on the `xsl:text` if EditorConfig respects that

I doubt EditorConfig is that smart :-)

> Alternatively we could say `<xsl:text>&#x0A;</xsl:text>`

That seems like the most straight forward solution